modules/core/shared/src/main/scala/freestyle/tagless/annotations.scala

```diff
@@ -19,13 +19,20 @@ package tagless
 
 import freestyle.tagless.internal._
 
+import scala.meta._
 import scala.annotation.{StaticAnnotation, compileTimeOnly}
 
 @compileTimeOnly("enable macro paradise to expand @tagless macro annotations")
-class tagless extends StaticAnnotation {
+class tagless(val stacksafe: Boolean) extends StaticAnnotation {
   import scala.meta._
 
-  inline def apply(defn: Any): Any = meta { taglessImpl.tagless(defn) }
+  inline def apply(defn: Any): Any = meta {
+    val stacksafe: Boolean = this match {
+      case q"new $_(${Lit.Boolean(ss)})" => ss
```

Comment:
This works for `@tagless(true)` but not for `@tagless(stacksafe = true)`; the compiler accepts both but won't generate the StackSafe tooling in the latter case. Given that naming boolean parameters is generally good practice, we should probably support it.

Reply:
Good stuff, the named parameter works in IDEA (though it took some wrangling than usual making the IDE happy - had to actually trigger a syntax error in the tag and then fix it to get it re-parsed ).